### DISCUSSION TITLE
Does ormar have ormar-only fields?

### DISCUSSION THREAD

**Discussion by sondrelg** [collerek/ormar#389] [Q&A] [answered]
Hi!

I have created a base model containing some standard fields (`id`, `created_at`, `updated_at`, etc.) that I use as the base for all my models. These are fields that I want in the database, but I don't want on pydantic models. So I basically just end up excluding these manually every time I call `Model.get_pydantic()`.

The (now to be deprecated) `pydantic_only` option seems like it would solve this issue for me, if it was reversed so I could do `ormar_only`. Does this exist in some form, or does anyone have tips for other ways to resolve this (tiny) issue? 🙂

**Comment by collerek** [comment, marked as answer]
There is no such thing as of now (btw. you cannot exclude primary keys).
If you set your fields to be populated in the database (like on update current timestamp), as I believe this is your use case, you can work around this by not including the fields in the model altogether. 

If you manage migrations with ormar and you need this column in the structure, then you can hack around this by also not including the fields in ormar, but adding those fields into the underlying sqlalchemy table (which is stored in Model.Meta.table), but note that this requires sqlalchemy core notation, not the orm one, so something like:

```python
column = sqlalchemy.Column(name, column_type, constraints etc.)
model.Meta.table.append_column(column)
```

Didn't test this but just out of my mind I think it shouldn't break anything 😁 😅

**Reply by sondrelg** [reply]
Ok, no worries, and thanks for the suggestion, I'll try it out 👏
